docs/troubleshooting.md

```diff
@@ -95,6 +97,15 @@ try to remove that file and restart the server.
 - clojure-lsp use clj-kondo to lint and cache in a `.clj-kondo/.cache` dir, try to remove that file as well if you think it's not linting correctly
 - If you have issues with macros, [double check your clj-kondo config](https://github.com/clj-kondo/clj-kondo/blob/master/doc/config.md#unrecognized-macros).
 
+### Missing `add code actions` using CoC and (neo)vim
```

Comment:
`add code actions`? I think you meant `add missing require`?

Reply:
ah yes